examples/openvino/aot/README.md

```diff
@@ -0,0 +1,115 @@
+# **Model Export Script for Executorch**
+
+This script allows users to export deep learning models from various model suites (TIMM, Torchvision, Hugging Face) to a openvino backend using **Executorch**. Users can dynamically specify the model, input shape, and target device.
+
+
+## **Usage**
+
+### **Command Structure**
+```bash
+python aot_openvino_compiler.py --suite <MODEL_SUITE> --model <MODEL_NAME> --input_shape <INPUT_SHAPE> --device <DEVICE>
+```
+
+### **Arguments**
+- **`--suite`** (required):
+  Specifies the model suite to use.
+  Supported values:
+  - `timm` (e.g., VGG16, ResNet50)
+  - `torchvision` (e.g., resnet18, mobilenet_v2)
+  - `huggingface` (e.g., bert-base-uncased). NB: Quantization and validation is not supported yet.
```

Comment:
why quant is not supported on hf models?

Reply:
There are no technical problems to quantize HF models. We already discussed some approach how to do it. However, we'd like to do this in the follow-up PR so as not to block the review and not make the PR too big.